public/index.html

```diff
@@ -5,7 +5,10 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="Web site created using create-react-app" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```

Comment:
as i know this web site wasn't created with create-react-app

Reply:
copy-paste from CRA???

Reply:
https://tech.onliner.by/2021/09/30/stack-overflow-sdelala-klaviaturu-s-knopkami-kopirovat-i-vstavit

Reply:
it was in initial project, I just used eslint --fix